### ISSUE TITLE
Step 2 contains extra "-"

### ISSUE THREAD

**Issue by DougBernhardt** [MicrosoftDocs/sql-docs#3000]
On step 2, there is an extra "-" at the end, correct?  Should read: wget -qO- https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f7b2f07a-852d-01b7-95b5-4e292cae2878
* Version Independent ID: 27712e05-1f53-3286-c78c-cb0623e53dfb
* Content: [Install azdata with installer on Linux - SQL Server big data clusters](https://docs.microsoft.com/en-us/sql/big-data-cluster/deploy-install-azdata-linux-package?view=sql-server-ver15#feedback)
* Content Source: [docs/big-data-cluster/deploy-install-azdata-linux-package.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/big-data-cluster/deploy-install-azdata-linux-package.md)
* Product: **sql**
* Technology: **big-data-cluster**
* GitHub Login: @MikeRayMSFT
* Microsoft Alias: **mikeray**

**Comment by msebolt** [comment]
@DougBernhardt  Thank you for the question.

The dash in this case refers to the file downloaded, see [this post](https://stackoverflow.com/questions/8045479/whats-the-magic-of-a-dash-in-command-line-parameters) for more information.

**Comment by Guillaume-Fourrat** [comment]
Well @msebolt perhaps that if @DougBernhardt reported this, there was an issue to begin with ? =)
I had reported it as well in PR https://github.com/MicrosoftDocs/sql-docs/pull/2901. On the basis that the current cmdline is NOT working (with error "gpg: can't open '–': No such file or directory"), and removing the dash IS working (returns "OK", and I double checked the key is added to the list).
Still, I reviewed the article you pointed, which is indeed relevant.
But cmd line is still failing, so what gives ? =)
Answer is as follow : the current cmdline uses ASCII 0x96 for dash (the "En Dash") instead of ASCII 0x2D.
If you delete the dash and retype it from keyboard's numpad, it should work.

But it also means the "-" isn't fully required as the default piping of stdout to apt-get seems to do the trick in this case.

I'll update my PR with the above.
